### PR TITLE
Make Background Transparent for Android

### LINKED ISSUE #86: Transparent Background on Android
The PNGs that are produced on Android for signatures have opaque, white backgrounds. On iOS, the background is transparent.

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #86. This PR maps the white background pixels for signature Bitmaps to transparent pixels. This way on both Android and iOS, the signature PNGs will have transparent backgrounds.

### CODE REVIEW

File: android/src/main/java/com/rssignaturecapture/RSSignatureCaptureView.java [hunk 1]
```diff
@@ -86,23 +82,26 @@ public RSSignatureCaptureView(Context context, SignatureCallback callback) {
...
-		}
-
-		// important for saving signature
+		// This view's pixels constitute the signature, so draw this view onto the signatureBitmap
 		final Canvas canvas = new Canvas(signatureBitmap);
 		this.draw(canvas);
 
+		// The signatureBitmap now contains the signature with a white background,
+		// so we will change it to a transparent background.
+		for (int x = 0; x < signatureBitmap.getWidth(); x++) {
```
Comment (john1jan) [inline android/src/main/java/com/rssignaturecapture/RSSignatureCaptureView.java]: there are two for loops. What about the performance of the component ? Did you do any performance test.

**Comment by john1jan** [comment]
@jedt any update on this

**Comment by john1jan** [comment]
@Hopding i have solved using a simple technique to setting transparent background. check #97

**Comment by Hopding** [comment]
I'm closing this in favor of https://github.com/RepairShopr/react-native-signature-capture/pull/97. It's been over a year since I opened this, so I doubt it's going to be merged anyhow.

**Pull request closed without merging**